client/src/Chat.jsx

```diff
@@ -1,44 +1,148 @@
-import React from "react";
- 
+import React from 'react';
+
 import {
-    ApolloClient,
-    InMemoryCache,
-    ApolloProvider,
-    useQuery,
-    gql
-  } from "@apollo/client";
-
-  const client = new ApolloClient({
-    uri: 'http://localhost:4000/',
-    cache: new InMemoryCache()
-  });
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  useQuery,
+  gql,
+  useMutation
+} from '@apollo/client';
+
+import { Container, Row, Col, FormInput, Button } from 'shards-react';
+
+const client = new ApolloClient({
+  uri: 'http://localhost:4000/',
+  cache: new InMemoryCache()
+});
+
+const GET_MESSAGES = gql`
+  query {
+    messages {
+      content
+      id
+      user
+    }
+  }
+`;
 
-  const GET_MESSAGES = gql`
-      query{
-        messages{
-          content
-          id
-          user
-        }
-      }`
-
-const Messages = ({user}) => {
-  const {data} = useQuery(GET_MESSAGES);
-  if(!data){
-    return null
+const POST_MESSAGES = gql`
```

Comment:
`POST_MESSAGE`, not messageS

Reply:
okay. i was following a tutorial on youtube , so i did exactly what the guy did. Will make the changes